frame/support/src/storage/mod.rs

```diff
@@ -716,6 +774,42 @@ pub struct PrefixIterator<T> {
 }
 
 impl<T> PrefixIterator<T> {
+	/// Creates a new `PrefixIterator`, iterating after `next_key` and filtering out keys that are
```

Comment:
I would personally name it `from_key` or `after_key` or `previous_key` instead of `next_key` because it is not the next fetched key.
And I would write it as a note.